deploy/run_with_summon.sh

```diff
@@ -4,15 +4,18 @@ set -xeuo pipefail
 . utils.sh
 
 # Clean up when script completes and fails
-function finish {
+finish() {
   # There is a TRAP in test_in_docker.sh to account for Docker deployments so we do not need to add another one here
   # Stop the running processes
   if [[ $RUN_IN_DOCKER = false && $DEV = false ]]; then
     announce 'Wrapping up and removing environment'
-    ./stop
-    cd ./kubernetes-conjur-deploy-$UNIQUE_TEST_ID && ./stop
+    repo_root_path=$(git rev-parse --show-toplevel)
+    "$repo_root_path/deploy/stop"
```

Comment:
what's this?

Reply:
why did we do changes for running with summon ?

Reply:
> what's this?

Gives us the head of the repo because we pushd in and out of directories so paths get messed up. This ensures that we are at the root of the repo

Reply:
but what's happening in this line? you just declare it without doing anything with it

Reply:
Why is nothing happening? Stop is being run

Reply:
See image attached 
<img width="1006" alt="Screen Shot 2020-08-18 at 8 44 11 AM" src="https://user-images.githubusercontent.com/19418506/90475566-58a35380-e130-11ea-80f9-70037152ad80.png">